rubygem-openscap/rubygem-openscap.spec

```diff
@@ -77,17 +77,6 @@ gem build %{gem_name}.gemspec
 gem install --local --install-dir .%{gem_dir} \
             --force %{SOURCE0} --no-rdoc --no-ri
 
-%check
-set +e
-rake test 2>&1 > test.output
```

Comment:
Don't remove the tests - just keep the `rake test` and drop all of the test.output parsing (i.e. revert most of b1c9a79).